README.md

```diff
@@ -1,21 +1,66 @@
 # Heat Pump Installer Chatbot Prototype
 
-This is just testing my git.
+This repository contains the **Heat Pump Installer Chatbot**, a prototype chatbot designed to assist heat pump installers with on-the-job queries via WhatsApp. Built with a focus on large language model (LLM) integration, the chatbot serves as a knowledge-enhancing tool, enabling installers to access relevant information and solutions in real time.
 
-## Setup
+## Key Features
```

Comment:
Would potentially advertise here that we can integrate with Streamlit as well as WhatsApp and then link it to the front-end we've made.